ts/routes/import-page/ImportLogPage.svelte

```diff
@@ -7,47 +7,46 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     import * as tr from "@generated/ftl";
 
     import Container from "$lib/components/Container.svelte";
+    import Row from "$lib/components/Row.svelte";
+    import TitledContainer from "$lib/components/TitledContainer.svelte";
 
-    import CloseButton from "./CloseButton.svelte";
     import DetailsTable from "./DetailsTable.svelte";
     import { getSummaries } from "./lib";
     import QueueSummary from "./QueueSummary.svelte";
 
     export let response: ImportResponse;
-    const result = response;
-    $: summaries = result ? getSummaries(result.log!) : [];
-    $: foundNotes = result?.log?.foundNotes ?? 0;
-    let closeButton: HTMLElement;
+    $: summaries = getSummaries(response.log!);
+    $: foundNotes = response.log?.foundNotes ?? 0;
+
+    const gutterBlockSize = 0.5;
+    const computedStyle = getComputedStyle(document.documentElement);
+    const rootFontSize = parseInt(computedStyle.fontSize);
+    // Container padding + Row padding + Row margin + TitledContainer padding
+    const bottomOffset = (3 * gutterBlockSize + 0.75) * rootFontSize;
```

Comment:
Not entirely happy with this fragile solution, but couldn't think of a viable alternative.